pkg/scheduler/metrics/util_windows.go

```diff
@@ -0,0 +1,32 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"time"
+)
+
+// SinceInSeconds gets the time since the specified start in seconds.
+func SinceInSeconds(start time.Time) float64 {
+	// On Windows, time.Now() is not as precise, 2 consecutive calls may return the same timestamp,
+	// resulting in 0 time delta / latency.
+	// See: https://github.com/golang/go/issues/8687
```

Comment:
The issue is closed.

Why is 0 delta a problem?

Reply:
Test assertions are expecting the latency to be non-zero: https://github.com/kubernetes/kubernetes/blob/70617042976dc168208a41b8a10caa61f9748617/pkg/scheduler/framework/runtime/framework_test.go#L2815

The metrics latencies are registered whenever certain operations complete (e.g.: ``RunPreFilterPlugins``), and ``SinceIsSeconds`` is used to calculate these deltas (e.g.: https://github.com/kubernetes/kubernetes/blob/70617042976dc168208a41b8a10caa61f9748617/pkg/scheduler/framework/runtime/framework.go#L607 ). But since ``time.Now()`` can return the same timestamp on Windows, the latency becomes 0.

These are just some measurements aggregated into histograms. Alternatively, we can skip the check ``if value <= 0`` mentioned above for Windows.

Reply:
if that's easy to achieve, that would make more sense.

Reply:
@alculquicondor , done. In the current version of the PR we're skipping the ``if value <= 0`` check.